airgun/views/virtwho_configure.py

```diff
@@ -19,13 +21,52 @@
 )
 
 
+class StatusIcon(GenericLocatorWidget):
+    """The status for virtwho configure can be: green, gray.
+
+    green: The virt-who report has not arrived within the interval,
+         which indicates there was no change on hypervisor
+    gray: The configuration was not deployed yet or the virt-who was
+        unable to report the status
+    """
+
+    def __init__(self, parent, locator=None, logger=None):
+        """Provide default locator value if it wasn't passed"""
+        if not locator:
+            locator = ".//span[contains(@class, 'virt-who-config-report-status')]"
+        Widget.__init__(self, parent, logger=logger)
+        self.locator = locator
+
+    @property
+    def color(self):
```

Comment:
I don't really like us obtaining generated color and doing decisions based on that. We should focus on message that this color carry.

Looking at [plugin code](https://github.com/theforeman/foreman_virt_who_configure/blob/master/app/helpers/foreman_virt_who_configure/configs_helper.rb#L44), I can see they add specific class based on information they want to provide. Can we rewrite that to provide `status` (not `color`) property and do decisions based on element classes?